scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -13,7 +13,7 @@
 class TestCircumcenters(object):
 
     def test_circumcenters(self):
-        tetrahedrons = np.array([
+        tets = np.array([
```

Comment:
Since a new algorithm is proposed in the PR, it may be easier to assess without variable name changes like this in the tests and so on.